### ISSUE TITLE
DualShock 4 gamepad does not get detected using Browser source

### ISSUE THREAD

**Issue by guihkx** [obsproject/obs-studio#6876]
### Operating System Info

Other

### Other OS

Arch Linux

### OBS Studio Version

27.2.4

### OBS Studio Log URL

https://obsproject.com/logs/u86rmCMy9to2keDp

### Expected Behavior

My DS4 gamepad should be detected once I press any of its buttons. It does work properly on Chrome 103 and Firefox 103.

### Current Behavior

The embedded Browser is not able to detect my DS4 gamepad.

### Steps to Reproduce

(I have a DS4 gamepad and I'm running OBS via Flatpak)

1. Add a browser source pointing to this URL: https://gamepadviewer.com/?p=1&s=8
2. Connect your gamepad
3. Select the Browser source and hit the 'Interact' button
4. Press any buttons on your gamepad

### Anything else we should know?

I thought this could be some Flatpak permission issue, so I used Flatseal to manually give these additional permissions to OBS:

- `--socket=session-bus`
- `--socket=system-bus`
- `--allow=bluetooth`

None of them made any difference. Plus, I have another Flatpak app called [jstest-gtk](https://flathub.org/apps/details/io.gitlab.jstest_gtk.jstest_gtk) that can detect my gamepad successfully, so my wild guess is that this issue lies on some missing API from CEF for Linux?

**Comment by guihkx** [comment]
I just discovered that all Chromium-based Flatpak apps have the exact same issue.

According to [this post](https://github.com/flathub/org.chromium.Chromium/issues/40#issuecomment-747266868) by a Flatpak collaborator, Chromium reads devices from [udev](https://en.wikipedia.org/wiki/Udev), but Flatpak does not expose this interface by default, due to ABI instability.

Then, another user pointed out [a workaround](https://github.com/flathub/org.chromium.Chromium/issues/40#issuecomment-825947686): Simply expose the `/run/udev` system path to Flatpak apps, and gamepads should be detected.

So, I went ahead and tried that with OBS:

```
flatpak override --user --filesystem=/run/udev com.obsproject.Studio
```

After restarting OBS, my gamepad is now detected successfully! Nice! 👌

Let's just hope it doesn't stop working...
